static/js/dashboard_routes.js

```diff
@@ -37,7 +38,9 @@ export function makeDashboardRoutes(browserHistory, store, onRouteUpdate) {
               <Route path="privacy" component={PrivacyTab} />
             </Route>
             <Route path="/terms_of_service" component={TermsOfServicePage} />
-            <Route path="/users/:username" component={UserPage} />
+            <Route path="/users" component={UserPage} >
```

Comment:
I could add an `IndexRedirect` here that would redirect to `/users/${username}` where username is the users username. Would that make sense?

The reason I wanted to have the `:username` route nested under `/users/` is that it simplifies sharing the prop injection pattern between `ProfilePage` and `UserPage` (both have routes underneath them and inject props into `this.props.children` to provide helper functions for dispatching actions and data from the store).